src/relax/transform/call_tir_rewrite.cc

```diff
@@ -33,7 +33,7 @@ namespace relax {
 
```

Comment:
nit: should we rename this file and pass to reflect that it handles both call_dps_packed and call_tir?

Reply:
I was thinking about this, but don't have a better name, any suggestions? if we change this, then the pass name CallTIRRewrite needs to be updated accordingly.

Reply:
DPSRewrite would be my choice but we can discuss and change it later.

Reply:
Thanks for the suggestion! We can discuss it later, it might need to get wider agreement for pass renaming.